airbyte-integrations/connectors/source-qonto/acceptance-test-config.yml

```diff
@@ -0,0 +1,20 @@
+# See [Source Acceptance Tests](https://docs.airbyte.io/connector-development/testing-connectors/source-acceptance-tests-reference)
+# for more information about how to configure these tests
+connector_image: airbyte/source-qonto:dev
+tests:
+  spec:
+    - spec_path: "source_qonto/spec.yaml"
+  connection:
+    - config_path: "secrets/config.json"
```

Comment:
@leilaballouard you should provide a `sample_config.json` which is a sample working json here and uses things like "REPLACEME" so someone can easily replicate for testing.